tests/periph_i2c/README.md

```diff
@@ -1,17 +1,50 @@
 # Periph I2C Test
 
 This application enables you to test all available low-level I2C functions
-using the RIOT shell.  Consult the 'help' shell command for available actions.
+using the RIOT shell. The `robot-test` feature requires a target board to be
+connected to the hardware reference board PHiLIP.
 
-## Background
+## Automated Test Setup
 
-Test for the low-level I2C driver.  Can be run manually or using automated test
-script for RobotFramework, run `make robot-test`.
+Before executing the automated test, the DUT (device under test) and reference
```

Comment:
DUT is in the test glossary, no need to write it out here

Reply:
yup